OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -230,7 +210,8 @@ public void Tick(Squad owner)
 
 			foreach (var a in owner.Units)
 			{
-				if (!ReloadsAutomatically(a) && !FullAmmo(a))
+				var ammoPool = a.TraitsImplementing<AmmoPool>().ToArray();
```

Comment:
ammoPool**s**.

Also, since we're dropping the `.All` in the methods the `.ToArray()` can go away.

Reply:
checkmark